bugbug/bugzilla.py

```diff
@@ -443,8 +443,8 @@ def calculate_maintenance_effectiveness_indicator(
     }
 
     logger.info(
-        "Calculating maintenance effectiveness indicator for the %s team from %s to %s",
-        team,
+        "Calculating maintenance effectiveness indicator for the %s teams from %s to %s",
+        ", ".join(teams),
```

Comment:
Nit: we could use "and": https://github.com/suhaibmujahid/bugbot/blob/a929257d756715933ec2672ac63da38ed66eeeaf/bugbot/utils.py#L181-L186